xcell/tests/test_mapper_k1000.py

```diff
@@ -0,0 +1,124 @@
+import numpy as np
+import xcell as xc
+import healpy as hp
+import os
+
+
+def get_config(w_stars=False):
+    if w_stars:
+        fname = 'xcell/tests/data/catalog_stars.fits'
+    else:
+        fname = 'xcell/tests/data/catalog.fits'
+    return {'data_catalog': fname,
+            'file_nz': 'xcell/tests/data/Nz_DIR_z0.1t0.3.asc',
+            'zbin': 0, 'nside': 32, 'mask_name': 'mask',
+            'e1_flag': 'bias_corrected_e1',
```

Comment:
because those are the names in the test fits files we use. The names come from the KV450 catalog. We could regenerate those files with new names, but every time we do that the size of the repo increases significantly.

Reply:
ok, but I think there're already `e1` and `e2` components in the catalog.

Reply:
yes, but they have different values that would not give rise to the maps we're testing with here. These maps are quite peculiar (they're something like `[0., 1., 2., 3., 0., 1., 2., 3, ...]`. I kind of like having something so specific.

We could open a separate issue for "homogenizing tests for shear maps"...

Reply:
I think it's a good idea, too.